pages/send/Send.tsx

```diff
@@ -91,25 +135,23 @@ export function Send() {
               className="flex flex-col gap-2 flex-1"
             >
               <KeyboardIcon className="text-muted-foreground" />
-              <Text numberOfLines={1}>Manual</Text>
+              <Text numberOfLines={1}>Manual / Contacts</Text>
```

Comment:
"Manual / Contacts" will not look good in the UI, I am not sure if we should even show text on the buttons. WoS uses simple icon buttons and works well